mobile/lib/features/student_management/presentation/pages/create_student_page.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../../domain/entities/student.dart';
+import '../bloc/student_bloc.dart';
+
+class AddClass extends StatefulWidget {
+  @override
+  _AddClassState createState() => _AddClassState();
+}
+
+class _AddClassState extends State<AddClass> {
+  TextEditingController _firstnameController = TextEditingController();
+  TextEditingController _lastnameController = TextEditingController();
+  TextEditingController _idController = TextEditingController();
+  TextEditingController _classroomIdController = TextEditingController();
+  StudentBloc _studentBloc;
+  int _levelSelected;
+  final List<int> levels = [
```

Comment:
Please, make sure we can understand that this is for test only. You can suffix it with "placeholder" or something similar